CONTRIBUTING.md

```diff
@@ -115,6 +116,27 @@ the project with Local File System input.
 resources/sample_md_reflections/lab1, resources/sample_md_reflections/lab2, resources/sample_md_reflections/lab3
 ```
 
+#### Missing Module Gensim or gensim.summarization With Windows
+
+If you run streamlit_web.py and are met with a `No module named Gensim` or similar
+error and are running windows, You can fix this problem by installing Microsoft Visual
+C++ 14.0+. You can do this by going to the [Visual Studio 2019 Downloads](https://visualstudio.microsoft.com/downloads/),
+scrolling down and expanding the `Tools for Visual Studio 2019` and clicking the download link for
+`Build tools for Visual Studio 2019`. 
+
+Once the installer is downloaded, launch it and select
+the `C++ build tools` option under `Desktop and Mobile`. After that select install at the bottom
+and wait. Note that it is a ~6GB download and may take some time. Once the C++ build tools are finished
+installing, the installer will ask to restart the computer to finish the installation.
+
+After your computer has finished installing the C++ build tools, you may need to reinstall the dependencies.
+To do this, delete the `Pipfile.lock` file in the root of the GatorMiner directory and reinstall the
```

Comment:
Does the user still need to delete `Pipfile.lock` since it's now updated? I'm not sure, ignore this if you think it's necessary.

Reply:
I'm not sure as I deleted it before testing just in case, but ill just add a section afterwards saying that it may be necessary on some systems.